src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -57,15 +61,21 @@ public void setup() {
     mockUserStore = Mockito.mock(UserStore.class);
     mockConvoStore = Mockito.mock(ConversationStore.class);
     mockMessageStore = Mockito.mock(MessageStore.class);
+    fakeUserID = UUID.randomUUID();
+    mockUser = Mockito.mock(User.class);
+    mockMessage = Mockito.mock(Message.class);
 
-    User fakeUser =
```

Comment:
Why did you omit the creation of the fake user?

Reply:
So I ran into uses because of the way that Mockito works. Essentially I was doing too much work by creating a fake user and fake message. Mockito allows you to return whatever you want when prompted. That's basically the logic behind Mockito.when().thenReturn. So essentially this allowed me to circumvent the "hardcoding" of creating fake users and all that.

Reply:
Okay, I'm actually working on that right now for the activity feed so that is very useful